schemas/com.snowplowanalytics.datamodeling/config/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+                                                                },
+                                                                "minItems": 0,
+                                                                "maxItems": 64
+                                                        }
+                                                },
+                                                "required": [
+                                                        "playbook",
+                                                        "dependsOn"
+                                                ],
+                                                "additionalProperties": false
+                                        },
+                                        "minItems": 1,
+                                        "maxItems": 64
+                                },
+                                "owners": {
+                                        "type": ["array", "null"],
+                                        "items": {
+                                                "type": "object",
+                                                "properties": {
+                                                        "name": {
+                                                                "type": "string",
+                                                                "minLength": 1,
+                                                                "maxLength": 64
+                                                        },
+                                                        "email": {
+                                                                "type": "string",
+                                                                "format": "email"
+                                                        }
+                                                },
+                                                "required": [
+                                                        "name",
+                                                        "email"
+                                                ],
+                                                "additionalProperties": true
+                                        },
+                                        "minItems": 0,
+                                        "maxItems": 8
+                                },
+                                "dagName": {
+                                        "type": ["string", "null"],
+                                        "minLength": 1,
+                                        "maxLength": 64
+                                },
+                                "sqlRunner": {
+                                        "type": ["string", "null"],
+                                        "pattern": "^(0|[1-9][0-9]{0,1})\\.(0|[1-9][0-9]{0,1})\\.(0|[1-9][0-9]{0,1})(-rc[0-9]{1,2}){0,1}$",
+                                        "maxLength":13
```

Comment:
I wouldn't impose a max length here either.

Reply:
We've just had a chat on another PR about maxLengths. It seems that having a maxLength is preferred because it is more explicit (for example, in an event/entity schema how to create the Redshift table) even if that maxLength has to be a huge overestimation (up to Redshift max of 65k but probably something like 4096 is fine?).

Reply:
Sure, I agree with that perspective from the point of view that when you're tracking data, there is a hard limitation somewhere (so for a string, redshift imposes a max length, and that should be represented in the schema).

When something genuinely doesn't have any such restriction it seems more wrong than right to specify one in the schema for me. Not something I care enough about to object strongly, so if we want to add a v long max length it's fine. Just philosophically seems odd.

Reply:
Philosophically I also agree and think that as this is a config, it should probably be removed.